src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -625,10 +624,16 @@ canvas.mapboxgl-canvas:focus {
       top: 16px;
     }
   }
-
 }
 
 #street-city-state-zip {
   font-weight: bold;
   padding-right: 28px;
-}
\ No newline at end of file
+}
+
+// Make focus styles on the Medallia 'Feedback' button consistent with focus styles in the survey
+.nebula_image_button.wcagOutline:focus {
+  outline: 2px solid rgb(249, 198, 66) !important;
```

Comment:
I tried to avoid using `!important`, but it didn't work without it. 

https://user-images.githubusercontent.com/6130520/109038683-9bbb2200-7691-11eb-8047-dcd9ad41da48.gif